developer-workflow/c-api.rst

```diff
@@ -53,14 +60,18 @@ Private names
 --------------
 
 Any API named with a leading underscore is also considered internal.
-There are two main use cases for using such names rather than putting the
-definition in ``Include/internal/`` (or directly in a ``.c`` file):
+There is currently only one main use case for using such names rather than
+putting the definition in ``Include/internal/`` (or directly in a ``.c`` file):
+
+* Internal helpers for other public API, which users should not call directly.
```

Comment:
```suggestion
* Internal helpers for other public APIs, which users should not call directly.
```

Missing `s` for plural (like most but not quite all words in English, the [well-established plural of API is APIs](https://english.stackexchange.com/questions/323688/what-is-the-plural-of-api)—this seems to be a pervasive mistake in this document generally)